### ISSUE TITLE
Can't install using pod

### ISSUE THREAD

**Issue by daviddliu** [SwiftyJSON/SwiftyJSON#236]
Hi, I'm using cocoapods 37.1. When I try to use my podfile to install (pod 'SwiftyJSON', '>=2.2.0'),  I get the error message 

<code>
[!] Unable to satisfy the following requirements:
- `SwiftyJSON (>= 2.2.0)` required by `Podfile`
</code>

Any idea what is happening?

**Comment by tangplin** [comment]
Podfile
```ruby
platform :ios, '8.0'
use_frameworks!

target 'MyApp' do
    pod 'SwiftyJSON', '~> 2.2.0'
end
```
Update the `Cocoapods`
```bash
sudo gem update
```
Update the `spec`
```
pod repo update
```
Install
```
pod install
```

**Comment by bradbernard** [comment]
Ran this and it still doesn't work... does it have to do with new CocoaPods version?

**Comment by bradbernard** [comment]
Podfile
```
platform :ios, '8.0'
use_frameworks!

pod 'Alamofire', '~> 1.2'
target 'Test' do
   pod 'SwiftyJSON', '~> 2.2.0'
end
```

$ pod install
```
Analyzing dependencies
Downloading dependencies
Using Alamofire (1.2.2)
Using SwiftyJSON (2.2.0)
Generating Pods project
Integrating client project

[!] CocoaPods did not set the base configuration of your project because your project already has a custom config set. In order for CocoaPods integration to work at all, please either set the base configurations of the target `Test` to `Pods/Target Support Files/Pods-Test/Pods-Test.debug.xcconfig` or include the `Pods/Target Support Files/Pods-Test/Pods-Test.debug.xcconfig` in your build configuration.

[!] CocoaPods did not set the base configuration of your project because your project already has a custom config set. In order for CocoaPods integration to work at all, please either set the base configurations of the target `Test` to `Pods/Target Support Files/Pods-Test/Pods-Test.release.xcconfig` or include the `Pods/Target Support Files/Pods-Test/Pods-Test.release.xcconfig` in your build configuration.
```

Then in my VC.swift
```
import SwiftyJSON --> No such module SwiftyJSON
```

$ pod --version

```
0.37.2
```

**Comment by DanGTZ** [comment]
Any update about this issue : having the same on my side

**Comment by craigpearce5** [comment]
I'm getting the same issue with xcode telling me no such module SwiftyJSON, however it still builds successfully.  That error also didn't show up until after I had cleaned the file following a pod update.

**Comment by gadget00** [comment]
I'm getting this error too.  First it said that it couldn't load the underlying module.  I deleted the .xcworkspace file and re-generated it with 'pod install' , and now it says 'no such module SwiftyJSON'
Also it doesn't build the project.

**Comment by khanhankkhan** [comment]
Where can I just download the SwiftyJson.framework instead of using pods. Pods never works

**Comment by craigpearce5** [comment]
@khanhankkhan From https://github.com/SwiftyJSON/SwiftyJSON

    Manually (iOS 7+, OS X 10.9+)

    To use this library in your project manually you may:

    for Projects, just drag SwiftyJSON.swift to the project tree
    for Workspaces, include the whole SwiftyJSON.xcodeproj

**Comment by craigpearce5** [comment]
@gadget00 I found that the issue was not related to SwiftyJSON but with Xcode.  That being said I didn't find a solution.  Try commenting out the SwiftyJSON code `//import SwiftyJSON`, does the error move to another line?  That's what was happening to me.

**Comment by zhigang1992** [comment]
@craigpearce5 Which version of Xcode/Swift are you using?

**Comment by craigpearce5** [comment]
@zhigang1992 As I mentioned in a previous comment, the error I was getting was most definitely related to either xcode or cocoapods.  The error xcode was giving me was non specific and moved to my 3rd import statement regardless of what that was importing.  For example, the error would transition to `import Alamofire` if I commented out `import SwiftyJSON`.  I'm not sure what fixed the error but it seemed to go away and I have submitted my app and been approved.
